### DISCUSSION TITLE
admin email address the same as a normal user?

### DISCUSSION THREAD

**Discussion by richardltc** [pocketbase/pocketbase#213] [Q&A] [answered]
Hi,

Is there any issue with creating a normal user, via the api that has the same email address as the admin?

I ask because I don't seem to be able to login as my admin now, after I create a normal user with the same email address as the admin. It's probably me, but I just thought I'd ask.

Thank you

Richard

**Comment by ganigeorgiev** [comment]
Users and Admins are separate entities each with their own table, so the user creation shouldn't have any effect on the existing admin accounts.

Could you clarify what actually is the issue? Could you check the error  response in the browser's "Inspect tool > Network tab" for the `/api/admins/auth-via-email` call?

**Comment by richardltc** [comment, marked as answer]
I'm getting a 400 invalid credentials: {"code":400,"message":"Failed to authenticate.","data":{}}

I must have somehow messed my password up. I could try again from scratch? or reset the admin password?

Richard

**Reply by ganigeorgiev** [reply]
You can start the application with `--debug` for more detailed error output in the console.

I'm not sure what is the causing your issue. Users are completely independent from the admins, so there is no reason why creating/updating a user would affect an admin account.

If you haven't setup your mail settings and you don't have access to the admin area, you may need to update the password manually using the sqlite3 cli tool to update the `_admins` record's `passwordHash` column (I'll add a console command in future version to simplify the process).

**Reply by richardltc** [reply]
Thanks, all sorted :)
